File: src/main/java/org/hypejet/hype/lang/PlayerDisconnectListener.java
package org.hypejet.hype.lang;

import net.minestom.server.event.player.PlayerDisconnectEvent;

final class PlayerDisconnectListener {
    public void listener(PlayerDisconnectEvent event, PlayerSettingsListener listener) {
        listener.locales.remove(event.getPlayer(), listener.locales.get(event.getPlayer()));
    }
}

File: src/main/kotlin/org/hypejet/hype/lang/PlayerDisconnectListener.kt
package org.hypejet.hype.lang

import net.minestom.server.event.player.PlayerDisconnectEvent

internal class PlayerDisconnectListener {
    fun listener(event: PlayerDisconnectEvent, listener: PlayerSettingsListener) {
        listener.locales.remove(event.player, listener.locales[event.player])
    }
}
